integration-tests/src/test/java/gov/uk/address/api/client/AddressApiClient.java

```diff
@@ -0,0 +1,97 @@
+package gov.uk.address.api.client;
+
+import uk.gov.di.ipv.cri.common.library.client.ClientConfigurationService;
+import uk.gov.di.ipv.cri.common.library.client.HttpHeaders;
+
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class AddressApiClient {
+    private final HttpClient httpClient;
+    private final ClientConfigurationService clientConfigurationService;
+    private static final String JSON_MIME_MEDIA_TYPE = "application/json";
+
+    public AddressApiClient(ClientConfigurationService clientConfigurationService) {
+        this.clientConfigurationService = clientConfigurationService;
+        this.httpClient = HttpClient.newBuilder().build();
+    }
+
+    public HttpResponse<String> sendPostCodeLookUpRequest(String sessionId, String postcode)
+            throws IOException, InterruptedException {
+        postcode = postcode.trim().replaceAll("\\s", "%20");
+        var request =
+                HttpRequest.newBuilder()
+                        .uri(
+                                new uk.gov.di.ipv.cri.common.library.util.URIBuilder(
```

Comment:
use an import and remove the fully qualified use of the URIBuilder
please change all occurences

Reply:
Done.